vahti/parser.py

```diff
@@ -28,6 +28,7 @@ def __init__(self, params=None, config=None):
 
     def start(self):
         self.session = ClientSession(
+            connector=TCPConnector(verify_ssl=False),
```

Comment:
We shouldn't make this a default. You can add it as a cli option in a different PR if you want.